spec/etl/ga/concerns/transform_path_spec.rb

```diff
@@ -11,9 +11,30 @@ class Dummy
     events_with_prefix = Events::GA.where("page_path ~ '^\/https:\/\/www.gov.uk'")
     expect(events_with_prefix.count).to eq 1
 
-    Dummy.new.remove_invalid_prefix
+    Dummy.new.format_events_with_invalid_prefix
 
     events_with_prefix = Events::GA.where("page_path ~ '^\/https:\/\/www.gov.uk'")
     expect(events_with_prefix.count).to eq 0
   end
+
+  context 'when an event exists with the same page_path after formatting' do
+    let!(:event2) { create(:ga_event, page_path: "/https://www.gov.uk/topics", process_name: 'views', unique_pageviews: 1, pageviews: 1) }
+
+    before(:each) do
+      create(:ga_event, page_path: "/topics", process_name: 'views', unique_pageviews: 100, pageviews: 200)
```

Comment:
I would use `FactoryGirl` traits to setup this factory:
I would like to express it this way:

```ruby
create(:ga_event, :views)
```